src/python/visclaw/data.py

```diff
@@ -223,7 +227,6 @@ def getframe(self,frameno,outdir=None,refresh=False):
             framesoln = self.framesoln_dict[key]
 
         return framesoln
-
```

Comment:
Need to add back the white-space here, really should have two lines in between function declarations.